.github/workflows/latest-install-test-react-native.yml

```diff
@@ -0,0 +1,45 @@
+name: Install test (React Native)
+
+on:
+  # Any change to these files
+  push:
+    paths:
+      - install-tests/react-native/**
+  # Every day at 9:00 CET
+  schedule:
+    - cron: "0 8 * * *"
+  # You can also activate this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      realm-version:
+        description: The version of Realm to install
+        required: true
+        default: latest
+      react-native-version:
+        description: The version of React Native to install
+        required: true
+        default: nightly
+
+jobs:
+  install:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14'
+      - name: Install npm v7
+        run: npm install -g npm@7
+      - name: Install React Native & Realm
+        run: install.sh ${{ github.event.inputs.react-native-version }} ${{ github.event.inputs.realm-version }}
+        working-directory: install-tests/react-native
```

Comment:
Since all commands should be run in this directory, I opted for moving this to [a `defaults` value](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#defaultsrun).

Reply:
Also good :)